cms/envs/common.py

```diff
@@ -96,6 +96,8 @@
     SUPPORT_SITE_LINK,
 
     CONTACT_EMAIL,
+
+    DISABLE_ACCOUNT_ACTIVATION_REQUIREMENT_SWITCH,
```

Comment:
It's not necessary in terms of workflows, but for some reason I had to specify it for devstack to start up (even the LMS).